### PR TITLE
added costs and changed all language to refer to REST API

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: docs/use-cases/etl.mdx [hunk 1]
```diff
@@ -3,11 +3,7 @@ title: "ETL / Data Extraction"
 description: "Data Extraction from Helicone"
 ---
 
-<Warning>
-  Please note that the GraphQL API is currently in its beta stage and may be subject to potential changes. We recommend using it with discretion.
-</Warning>
-
-Our [GraphQL API](graphql/getting-started) provides a robust solution for Extracting, Transforming, and Loading (ETL) data from Helicone into your personal data warehouse.
+Our [RES API](rest) provides a robust solution for Extracting, Transforming, and Loading (ETL) data from Helicone into your personal data warehouse.
```
Comment (ScottMktn) [inline docs/use-cases/etl.mdx:6]: REST*

File: docs/use-cases/etl.mdx [hunk 2]
```diff
@@ -24,4 +20,4 @@ Here are some key advantages of our ETL API compared to traditional logging tool
 
 At present, we do not offer any pre-built connectors. However, we are receptive to suggestions and committed to meeting our users' needs. If there is a specific connector you would like us to develop, please let us know. You can reach us on [Discord](https://discord.com/invite/zsSTcH2qhG) or submit a [Github issue](https://github.com/Helicone/helicone/issues).
 
-In the interim, you can design your own ETL process using our [GraphQL API](graphql/getting-started).
\ No newline at end of file
+In the interim, you can design your own ETL process using our [GraphQL API](graphql/getting-started).
```
Comment (ScottMktn) [inline docs/use-cases/etl.mdx:23]: Is this supposed to be GraphQL API?

File: docs/use-cases/etl.mdx (hunk 1 above)
Comment (chitalian) [inline docs/use-cases/etl.mdx:6]:
```suggestion
Our [REST API](rest) provides a robust solution for Extracting, Transforming, and Loading (ETL) data from Helicone into your personal data warehouse.
```

File: docs/use-cases/etl.mdx (hunk 2 above)
Comment (chitalian) [inline docs/use-cases/etl.mdx:23]:
```suggestion
In the interim, you can design your own ETL process using our [REST API](rest).
```

**Pull request merged**